src/python/WMCore/Lexicon.py

```diff
@@ -727,11 +725,5 @@ def getStringsBetween(start, end, source):
 
 def getIterMatchObjectOnRegexp(filePath, regexp):
     with io.open(filePath, 'r', encoding='utf8', errors='ignore') as f:
-        try:
```

Comment:
I honestly do not know what I decided to implement this with `mmap`, after all, whatever files we read, we load it into memory...
Still, just a note because I do not know whether its removal can cause issues in the near future.